Dockerfile

```diff
@@ -1,18 +1,20 @@
 FROM ubuntu:bionic
 MAINTAINER Evan Sultanik <evan.sultanik@trailofbits.com>
 
-RUN apt-get -y update \
- && apt-get install -y \
-    cmake \
-    git \
-    gnupg \
-    ninja-build \
-    python3-pip \
-    python3.7 \
-    wget
+RUN apt-get -y update && apt-get install -y cmake nlohmann-json-dev gnupg wget ninja-build python3.7 python3-pip git
+
+RUN python3.7 -m pip install pytest
 
 RUN wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add - && echo "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-7 main" | tee -a /etc/apt/sources.list && apt-get -y update && apt-get install -y llvm-7 clang-7
 
+WORKDIR /
+
+RUN git clone https://github.com/RoaringBitmap/CRoaring.git
+
+WORKDIR /CRoaring
+
+RUN mkdir -p build && cd build && cmake .. && make && make install
+
```

Comment:
These changes shouldn't be needed, now that the CRoaring and nlohmann-json libraries have been vendored.